frontend/src/common/components/naviconsidebar/NavIconHomeButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, IconButton, Tooltip } from "@mui/material";
```

Comment:
Just personal preference, but I prefer "NavIconSidebar" to be the directory name over "naviconsidebar".

Reply:
renamed to `NavIconSideBar`